{"uuid": "8c2e2034-d766-4f3b-b212-fadd181d2199", "children": ["04b3f920-8a5a-4366-9473-599d8ee8a600", "da63a989-1664-4923-8786-b7ccb24c9f51", "6f58cdd0-8247-46f6-8e30-8d22914c11c1", "46c95740-e9d1-4a71-aa7d-73bdb295dd79"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1586841309288, "stop": 1586841309288}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "NameError: name 'mail_pass' is not defined\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 707, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/sonic/Project/Python/Pytest/scripts/test_case.py\", line 41, in teardown_class\n    EmailHandler().send_email()\n  File \"/Users/sonic/Project/Python/Pytest/uti/EmailHandler.py\", line 59, in send_email\n    smtp_obj.login(mail_user, mail_pass)\n"}, "start": 1586841309956, "stop": 1586841318023}], "start": 1586841309288, "stop": 1586841318023}